### PR TITLE
[2.5] Fix gke-operator charts

### PR DESCRIPTION
**Pull request by cmurphy**
Sync 1.0.200 with 2.5.9 branch, move current 1.0.200 package to 1.0.300, bump gke-operator to latest for 1.0.300.

### CODE REVIEW

File: charts/rancher-gke-operator/1.0.200/Chart.yaml [hunk 1]
```diff
@@ -5,4 +5,4 @@ home: https://github.com/rancher/gke-operator
 sources:
   - "https://github.com/rancher/gke-operator"
 version: 1.0.200
-appVersion: v1.1.1-rc5
+appVersion: v1.0.2-rc2
```
Comment (cbron) [inline charts/rancher-gke-operator/1.0.200/Chart.yaml]: We'll need to remember to bump this to final release in both branches.

File: charts/rancher-gke-operator/1.0.300/questions.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.8-patch1
```
Comment (cbron) [inline charts/rancher-gke-operator/1.0.300/questions.yaml]: Should be 2.5.10-patch1, as Rancher codebase won't support this version until then. We don't want rancher v2.5.9 eventually getting this from release-v2.5 when 2.5.10 goes out.

**Review by cbron — CHANGES_REQUESTED** [review]
Minor change. Once 2.5.9 goes into release-v2.5 this may conflict, we'll need to pull 2.5.9 branch in here.

File: charts/rancher-gke-operator/1.0.300/questions.yaml (hunk 2 above)
Comment (cmurphy) [reply charts/rancher-gke-operator/1.0.300/questions.yaml]: done

**Review by thedadams — DISMISSED** [review]

**Review by thedadams — DISMISSED** [review]

**Comment by jiaqiluo** [comment]
@cmurphy  Can you update the version in https://github.com/rancher/rancher/issues/33528 once this PR is merged?

**Review by cbron — DISMISSED** [review]
lgtm, note that the 1.0.200 min version changed.

File: charts/rancher-gke-operator/1.0.200/Chart.yaml (hunk 1 above)
Comment (cmurphy) [reply charts/rancher-gke-operator/1.0.200/Chart.yaml]: Will update after https://github.com/rancher/system-charts/pull/503

**Review by thedadams — APPROVED** [review]

**Review by cbron — APPROVED** [review]

**Pull request merged**